packages/editor/src/extensions/react/selection-based-react-node-view.tsx

```diff
@@ -90,9 +90,7 @@ export class SelectionBasedNodeView<
     forwardRef?: ForwardRef
   ): React.ReactElement<unknown> | null {
     if (!this.options.component) return null;
-    const isSelected =
-      this.editor.isEditable &&
-      this.isSelectedNode(this.editor.view.state.selection);
+    const isSelected = this.isSelectedNode(this.editor.view.state.selection);
```

Comment:
Instead of enabling selection for all node views, we should make it an option. That way we can specifically enable selection in readonly mode for each node view.